Flow Actions/Assign Role/assignRole.js

```diff
@@ -0,0 +1,20 @@
+(function execute(inputs, outputs) {
+
+    var user = inputs.user_id;
+    var roleSysId = inputs.role_sys_id;
+
+    var userGR = new GlideRecord("sys_user");
+    userGR.addQuery('user_name', user);
+    userGR.query();
+
+    if (userGR.next()) {
+        var userSysId = userGR.getValue('sys_id');
+    }
+
+    var userRole = new GlideRecord('sys_user_has_role');
+    userRole.initialize();
+    userRole.setValue('user', userSysId);
+    userRole.setValue('role', roleSysId);
```

Comment:
As mentioned above, decide if role sys_id really makes sense or not.

Reply:
Hi Lacah, As to assign a role to a particular user we need to create a record for that user in the "sys_user_has_role" table. In this table the user and role field are of reference type, and to set the value of reference field we need sys_id. This is the reason why I am using the sys_id for the role and user. In the input I am taking the user name and the role as input. I have attached the screenshot for the same in the readme.md file.

Reply:
Thanks Lacah, for you valuable suggesstion. I will update this and will raise the new PR for this.